src/gui/dialogs/editor/custom_tod.cpp

```diff
@@ -197,12 +199,19 @@ std::string custom_tod::to_wesnoth_path(std::string abs_path, std::string asset_
 	} else if (is_in_addon_dir) {
 		rel_path = abs_path.erase(0, addon_asset_dir.size()+1);
 	} else {
-		// image outside wesnoth's directories
-		// copy to addon's images directory
+		// file outside, ask the user if they want to copy
+		const std::string& message
+			= _("File outside Wesnoth's data dirs. Do you wish to copy it in your add-on?");
```

Comment:
"into your add-on"

Maybe also "This file is outside".

Reply:
Added.